docs-site/content/guide/docsearch.md

```diff
@@ -74,10 +86,10 @@ Example: all extracted records on the page will have a `language` attribute of `
 
 The easiest way to run the scraper is using Docker.
 
-1. [Install Docker](https://docs.docker.com/get-docker/)
-2. [Install jq](https://stedolan.github.io/jq/download/)
-3. [Run Typesense](./install-typesense.md)
-4. Create a `.env` file with the following contents, replacing them with the correct values for your particular situation:
+1. [Install Docker.](https://docs.docker.com/get-docker/)
+1. [Install `jq`.](https://stedolan.github.io/jq/download/)
```

Comment:
putting all of the numbers to 1 is deliberate because writing out each individual number is an antipattern in markdown